trunner/target/armv7m7.py

```diff
@@ -69,7 +69,9 @@ def build_test(self, test: TestOptions) -> Callable[[TestResult], TestResult]:
 
             if test.bootloader.apps:
                 app_loader = PloPhoenixdAppLoader(
-                    dut=self.dut, apps=test.bootloader.apps, phoenixd=Phoenixd(directory=self.bin_dir())
+                    dut=self.dut,
+                    apps=test.bootloader.apps,
+                    phoenixd=Phoenixd(directory=self.root_dir() + test.shell.path),
```

Comment:
Assuming that `Path` type will be used.
```suggestion
                    phoenixd=Phoenixd(directory=self.root_dir() / test.shell.path),
```